### DISCUSSION TITLE
weird fabric crashes with certain mods

### DISCUSSION THREAD

**Discussion by dxcvvxd** [FabricMC/fabric-api#3371] [Player support] [answered]
### Minecraft Logs

```txt
.
```

### Description

so im using fabulously optimized and i've also added a few mods ontop of it and some mods after updating to 1.20.2 have stopped working and straight up crash the game because of some mixin errors, and many of them are caused by `fabric-<something>-v1`
the mods in question are these (also nochatreports crash of which was caused by fabric-network-v1 or something like that)
![image](https://github.com/FabricMC/fabric/assets/77129815/414578d7-7700-4954-9b53-969334ac1869)
i have attached logs for every single mod if you want more details. im not sure if its caused by conflicts with other mods and fabric api as well

im running prismlauncher latest on artix linux

logs:
1. better rp runtime
[latest.log](https://github.com/FabricMC/fabric/files/12838079/latest.log)
2. chunkumulator
[latest.log](https://github.com/FabricMC/fabric/files/12838083/latest.log)
3. fastload
[latest.log](https://github.com/FabricMC/fabric/files/12838084/latest.log)
4. hold that chunk
[latest.log](https://github.com/FabricMC/fabric/files/12838085/latest.log)
5. mouse wheelie
[latest.log](https://github.com/FabricMC/fabric/files/12838086/latest.log)
6. memory leak fix
[latest.log](https://github.com/FabricMC/fabric/files/12838087/latest.log)
7. tiefix
[latest.log](https://github.com/FabricMC/fabric/files/12838089/latest.log)
8. toomanyplayers
[latest.log](https://github.com/FabricMC/fabric/files/12838090/latest.log)

**Comment by Poopooracoocoo** [comment, marked as answer]
ViaFabricPlus is currently incompatible with No Chat Reports. https://github.com/ViaVersion/ViaFabricPlus/issues/261

You're using the wrong version of Mouse Wheelie.

The other mods aren't available for Minecraft 1.20.2. Ever since Minecraft 1.19.3, "dot" releases have been breaking. https://help.minecraft.net/hc/en-us/articles/9971900758413

Make sure to filter by Minecraft version when you download mods on Modrinth and CurseForge. I also recommend downloading mods through your launcher, which is Prism Launcher, as it'll automatically filter by modloader and game version. 

I do not recommend changing the Minecraft version of your custom instance, or at least when doing so you should know to check for incompatibilities yourself by reading the logs and by checking on Modrinth and CurseForge.

**Comment by dxcvvxd** [comment]
the thing is, i've always did everything you said i should do and only now it broke. this is why i was concerned.

**Comment by megamax20072007** [comment]
https://github.com/orgs/FabricMC/discussions/3385

**Reply by Poopooracoocoo** [reply]
What does your issue have to do with this?
